src/pages/api/assistant/chat/[id].ts

```diff
@@ -41,17 +47,23 @@ export const GET: APIRoute = async ({ locals, request, params }) => {
   const message = urlParams.get("message");
   const idType = urlParams.get("type");
 
-  if (!idType) {
+  if (!idType && id !== "all") {
     return new Response("Missing type. Valid values are 'file' or 'note'", {
       status: 400,
     });
   }
 
   let conversation = await db.query.conversations.findFirst({
-    where: and(
-      eq(conversations.userId, currentUser.id),
-      or(eq(conversations.fileId, id), eq(conversations.noteId, id))
-    ),
+    where:
+      id !== "all"
```

Comment:
I feel like we should follow the structure we had in the document and split the endpoints with a note/file and the endpoints without